beetmoverscript/data/beetmover_task_schema.json

```diff
@@ -79,9 +79,12 @@
                                 "items": {
                                     "type": "string"
                                 }
+                            },
+                            "zipExtract": {
+                                "type": "boolean"
                             }
                         },
-                        "required": ["taskId", "taskType", "paths", "locale"]
```

Comment:
I presume you remove 'locale' to fit the maven jobs right? The old beetmover jobs are still having the `locale`. Maybe it'd be worth adding it as optional.

Reply:
Yup, that's what this change is about. `locale` is now optional

Reply:
👍